shell/client/accounts/account-settings.js

```diff
@@ -305,7 +304,7 @@ Template.sandstormAccountSettings.events({
 
   "click button.delete-account-real": function (ev, instance) {
     instance._deleteSubmitting.set(true);
-    Meteor.call("deleteOwnAccount", instance._deleteFeedback.get(), function (err) {
+    Meteor.call("deleteOwnAccount", instance.$(".feedback").val(), function (err) {
```

Comment:
FWIW we also seem to use jQuery in [shell/client/grain-client.js](https://github.com/sandstorm-io/sandstorm/blob/master/shell/client/grain-client.js#L389..L391)